### DISCUSSION TITLE
Near-term vetting: Wind

### DISCUSSION THREAD

**Discussion by RahelMA** [pik-piam/mrremind#543] [Near-term validation]
**Variable: `Capacity|Electricity|Wind`**

**Source: GWEC | GLOBAL WIND REPORT 2026, IRENA 2025**

* regional data
* GWEC industry perspective for expected installations of new capacity for the next five
years (2025:2030). The outlook is based on input from regional wind associations, government targets, tender results, announced
auction plans, available project pipeline, and input from industry experts and GWEC members.

Assumptions + data preparation:

* fast expanding technology: new power plants can be ramped up rather quickly, allowing for more flexibility on maximum values
* regional definition varies among models, therefore more generous bounds and no red bounds for 2025 and 2030
* historical values reflect less uncertainty 

Threshold definitions:

**2020**
 _historical value, source IRENA_
* global  `min_red`: -10% ; regional  `min_red`: -40%
* global `min_yel`:  -5%    ; regional  `min_yel`: -20%
* global `max_yel`: +5%   ; regional  `max_yel`: +20%
* global `max_red`: +10% ; regional  `max_red`: +40%

**2025**
 _IRENA capacity for 2024 and GWEC capacity addition for 2025_
* global  `min_red`: -10% (IRENA2024+ CapAdd25 * 0.375) 
* global `min_yel`:  -5% (IRENA2024+ CapAdd25 * 0.75)     ; regional  `min_yel`: -40% (IRENA2024+ CapAdd25 * 0.375) 
* global `max_yel`: +5% (IRENA2024+ CapAdd25 * 1.5)   ; regional  `max_yel`: +40% (IRENA2024+ CapAdd25 * 2) 
* global `max_red`: +10%  (IRENA2024+ CapAdd25 * 2) 

**2030**
 _IRENA capacity for 2024 and GWEC capacity addition for 2025:2030_
* global  `min_red`: -10% (IRENA2024+ CapAdd25:30 * 0.375) 
* global `min_yel`:  -5% (IRENA2024+ CapAdd25:30 * 0.75)     ; regional  `min_yel`: -40% (IRENA2024+ CapAdd25:30 * 0.375) 
* global `max_yel`: +5% (IRENA2024+ CapAdd25:30 * 1.5)   ; regional  `max_yel`: +40% (IRENA2024+ CapAdd25:30 * 2) 
* global `max_red`: +10%  (IRENA2024+ CapAdd25:30 * 2) 

**Data Sources**
GWEC: [2026 GWEC Report](https://www.gwec.net/reports/globalwindreport)
IRENA: https://www.irena.org/Publications/2025/Mar/Renewable-capacity-statistics-2025 (2025-08-01)
EMBER: https://ember-climate.org/data-catalogue/yearly-electricity-data/ (2024-04-06)

**Change log**
- Update 08.2025: move base-year to 2024
- Update 04.2026: use new 2026 GWEC report

**Comment by lecfab** [comment]
GEM
- ignores smaller capacity projects < 10 MW
- global wind tracker updated in 2025
- values for the future consider that all the projects are successful (in mrremind)

EMBER
- does not distinguish offshore from onshore
- data until 2023

IRENA
- same values as EMBER 
- separated onshore offshore

GWEC
- data read from pdf charts
- only some regions
- recent capacity additions
